### ISSUE TITLE
 code breaks on Chrome on the first load.

### ISSUE THREAD

**Issue by jjhesk** [zurb/orbit#104]
I finally noticed that the code will break on Chrome if the pictures are first loading from the server,.. try many times on the official and my page too. It happens all the time.

Try to test it on simulator on chrome

**Comment by rakke** [comment]
I've also noticed this problem http://www.zurb.com/playground/orbit-jquery-image-slider in Chrome 16.0.912.75 on win 7. The page breaks, and no slides show. 
screenshot at https://github.com/rakke/Bug-report-attachements/raw/master/Orbit%20jQuery%20Image%20Slider%20Plugin%20from%20ZURB%20-%20ZURB%20Playground%20-%20ZURB.png

**Comment by jjhesk** [comment]
Yes, this is the exactly what i got from ORBITZ. Please fix it!!!

**Comment by lakesta** [comment]
https://github.com/zurb/orbit/issues/51#issuecomment-2231832

Check that - the fix was for something else but worked to fix this for me.
